task/02-numbers-tasks.js

```diff
@@ -202,7 +208,15 @@ function roundToPowerOfTen(num, pow) {
  *   17 => true
  */
 function isPrime(n) {
```

Comment:
It’s not an optimal solution. You use too many and too specific conditions. Please try to find a consecutiveness. It’s solvable with 2 conditions and 1 loop

Reply:
I had corrected but i don't think it seems to be more readable...

Reply:
Give one more try. You can use internet to find a smaller solution